docs/3to4.md

```diff
@@ -1,61 +1,61 @@
 ## Breaking Changes from Jedis 3 to Jedis 4 and Hints to Migrate
 
-- Jedis and JedisPool constructors with a String parameter without any int parameter will only support a URL or URI string.
+- Jedis and JedisPool constructors with a String parameter, and no int parameter only support a URL or URI string.
   - Jedis(String)
   - JedisPool(String)
   - JedisPool(String, SSLSocketFactory, SSLParameters, HostnameVerifier)
   - JedisPool(GenericObjectPoolConfig<Jedis>, String)
 
-- In xrange or xrevrange methods with StreamEntryID as parameters,
-  sending `null` for both start and end parameters would raise conflict.
+- [XRANGE](https://redis.io/commands/XRANGE) and [XREVRANGE](https://redis.io/commands/xrevrange) methods with StreamEntryID as parameters,
+  sending `null` for both start and end parameters will raise conflicts.
 
-  Casting the `null`s to StreamEntryID (`(StreamEntryID) null`) would resolve the issue.
+  Casting the `null`s to StreamEntryID (`(StreamEntryID) null`) resolves this issue.
 
-- Many SortedSet methods will return Java `List` instead of `Set`. [Listed here](3to4-zset-list.md)
+- Many SortedSet methods return Java `List` instead of `Set`. [Listed here](3to4-zset-list.md)
 
-- Many methods now returns primitive values (`long`/`boolean`/`double` instead of `Long`/`Boolean`/`Double`). [Listed here](3to4-primitives.md)
+- Many methods return primitive values (`long`/`boolean`/`double` instead of `Long`/`Boolean`/`Double`). [Listed here](3to4-primitives.md)
 
-- BinaryJedis and BinaryJedisCluster classes are removed.
+- BinaryJedis and BinaryJedisCluster classes have been removed.
 
   Methods of these classes are available in Jedis and JedisCluster classes respectively.
 
-- Client and BinaryClient classes are removed.
+- Client and BinaryClient classes have been removed.
 
-- Commands interfaces are restructured. In other words, `redis.clients.jedis.commands` package is reimplemented.
+- Commands interfaces were restructured, meaning `redis.clients.jedis.commands` package was reimplemented.
 
-- Return type of Jedis.shutdown() is now `void`.<br>
-  Previously, in ideal scenario, it would always return null.
+- The return type of Jedis.shutdown() is now `void`.<br>
+  Previously, ideally, it would return null.
 
-- ShardedJedisPool, Sharded, ShardedJedis, BinaryShardedJedis, ShardInfo, JedisShardInfo classes are removed.
+- ShardedJedisPool, Sharded, ShardedJedis, BinaryShardedJedis, ShardInfo, JedisShardInfo classes have been removed.
   - Introduced JedisSharding class.
 
-      Earlier codes without the use of "name" and "weight" (in ShardInfo/JedisShardInfo) are easily transferable new class.
+      Earlier code without the use of "name" and "weight" (in ShardInfo/JedisShardInfo) are easily transferable new class.
```

Comment:
Earlier code without the use of "name" and "weight" (in ShardInfo/JedisShardInfo) are easily transferable to new class.